src/net_processing.cpp

```diff
@@ -4499,6 +4515,16 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
         const uint256& hash = peer->m_wtxid_relay ? wtxid : txid;
         AddKnownTx(*peer, hash);
 
+        if (m_tx_for_private_broadcast.Remove(ptx)) {
```

Comment:
bc05ddc7b9369cf263bf88d0a773ca853d3c9af5 maybe I'm missing something, but should we not also adjust `m_private_broadcast_connections_to_open` when we get our privately broadcast tx back over regular relay? In my first local test, all 5 onion connections were completed before I got the tx back. But if the very first private connection relays the tx around and we get it back, do we still need to bother with the next 4?

Reply:
Yes, good idea. Implemented.